### PR TITLE
add cloudbuild artifacts

### PR DESCRIPTION
**Pull request by prasita123**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -14,6 +14,21 @@ steps:
...
+    # _GIT_TAG is not a valid semver, we use CI=1 instead
+    # - VERSION=$_GIT_TAG
+    - CI=$_CI
+    - PULL_BASE_REF=$_PULL_BASE_REF
+    - DOCKER_REGISTRY=$_DOCKER_REGISTRY
+    - DOCKER_IMAGE_PREFIX=$_DOCKER_IMAGE_PREFIX
+    - GCS_LOCATION=$_GCS_LOCATION
+    - LATEST_FILE=markers/${_PULL_BASE_REF}/latest-ci.txt
+    args:
+    - cloudbuild-artifacts
```
Comment (nckturner) [inline cloudbuild.yaml:31]: We need to add this target in our Makefile, either in this PR or a separate PR that is merged first.

Comment (prasita123) [reply cloudbuild.yaml:31]: added in https://github.com/kubernetes/cloud-provider-aws/pull/439

**Comment by prasita123** [comment]
adding in https://github.com/kubernetes/cloud-provider-aws/pull/439

**Pull request closed without merging**